src/tools/check/src/main.rs

```diff
@@ -133,6 +135,33 @@ impl Check {
                     }
                 };
 
+                match _config.engine.translation_layer {
```

Comment:
If you use variable, then its name shouldn't be `_` prefixed change name to `config`